mycity/mycity/intents/sailing_weather_intent.py

```diff
@@ -0,0 +1,75 @@
+"""
+Alexa intent used to find the sailing weather conditions 
+on the Charles River
+"""
+
+import requests
+
+from mycity.mycity_response_data_model import MyCityResponseDataModel
+
+
+COMMUNITY_SAILING = "https://api2.community-boating.org/api/flag"
+
+
+def get_sailing_weather_intent(mycity_request):
+    """
+    Retrieves the sailing conditions from Community Boating's API
+    and provides it to the user in an easily digestible format.
+    
+    :param mycity_request: MyCityRequestDataModel object
+    :return: MyCityResponseDataModel object
+    """
+    print(
+        '[method: get_sailing_weather_intent]',
+        'MyCityRequestDataModel received:',
+        str(mycity_request)
+    )
+    mycity_response = MyCityResponseDataModel()
+    
+    r = requests.get("https://api2.community-boating.org/api/flag")
+    
+    mycity_response.output_speech = get_sailing_conditions(r.text)
+    mycity_response.session_attributes = mycity_request.session_attributes
+    mycity_response.card_title = mycity_request.intent_name
+    mycity_response.reprompt_text = None
+    mycity_response.should_end_session = True
+    return mycity_response
+
+def get_sailing_conditions(api_response):
+    """
+    Checks for validity and parses the string returned from the API
+    
+    :param api_response: string indicating current sailing conditions
+    :return: string with message detailing current sailing conditions
+    """
+    server_flag_msg = api_response  # Ex:     var FLAG_COLOR = "Y"
+    if isinstance(server_flag_msg,str) and len(server_flag_msg) >= 19:
+        trimmed_flag_msg = server_flag_msg[18]  # Will produce a letter: G,Y,R,C
```

Comment:
I think we can make the intention here more clear then just indexing into the string. How about using Python's string split or even a regular expression?